lib/type_check/type_error/default_formatter.ex

```diff
@@ -280,7 +280,7 @@ defmodule TypeCheck.TypeError.DefaultFormatter do
     name = Map.get(s, :name, "#Function<...>")
     function_with_arity = IO.ANSI.format_fragment([:white, "#{name}/#{Enum.count(val)}", :red])
     param_spec = s.param_types |> Enum.at(index) |> TypeCheck.Inspect.inspect_binary(inspect_type_opts())
-    arguments = val |> Enum.map(&inspect/1) |> Enum.join(", ")
+    arguments = val |> Enum.map(fn val -> inspect(val, inspect_value_opts()) end) |> Enum.join(", ")
```

Comment:
I think you can still use capture syntax (unless you have personal preference)
```suggestion
    arguments = val |> Enum.map(&inspect(&1, inspect_value_opts())) |> Enum.join(", ")
```